ui/src/data_explorer/containers/DataExplorer.tsx

```diff
@@ -148,55 +149,32 @@ export class DataExplorer extends PureComponent<Props, State> {
   }
 
   public async componentDidMount() {
-    const {
-      source,
-      loadDE,
-      timeRange,
-      dashboards,
-      autoRefresh,
-      queryDrafts,
-      handleGetDashboards,
-    } = this.props
-    const {query} = qs.parse(location.search, {ignoreQueryPrefix: true})
+    const {loadDE, timeRange, autoRefresh, queryDrafts} = this.props
+    const {query, script} = this.queryString
```

Comment:
I somewhat expected `queryString` to a string

Reply:
its a parsed query string